src/algebraic_geometry/presheafed_space.lean

```diff
@@ -0,0 +1,173 @@
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+import category_theory.instances.Top.presheaf
+
+universes v u
+
+open category_theory
+open category_theory.instances
+open category_theory.instances.Top
+open topological_space
+
+variables (C : Type u) [𝒞 : category.{v+1} C]
+include 𝒞
+
+namespace algebraic_geometry
+
+structure PresheafedSpace :=
+(X : Top.{v})
+(𝒪 : X.presheaf C)
+
+variables {C}
+
+namespace PresheafedSpace
+
+instance : has_coe_to_sort (PresheafedSpace.{v} C) :=
```

Comment:
Is there a benefit in making `forgetful_functor.obj` the coercion? If so, this should maybe be a temporary local instance, until the category and functor have been set up.